lib/win32/screenshot/image.rb

```diff
@@ -27,6 +27,17 @@ def initialize(blob, width, height)
       # @raise [RuntimeError] when _file_path_ is not with the supported output {FORMATS} extension.
       def write(file_path)
         raise "File already exists: #{file_path}!" if File.exists? file_path
+        write_file file_path
+      end
+      
+      #Writes image to disk, writing over existing copy if it exists.
+      def write!(file_path)
+        write_file file_path
+      end
+      
+      private
+      
+      def write_file(file_path)
```

Comment:
No need to have this separate method here. Just inline it to `#write!` and call that method from `#write`. Makes perfect sense to me.